quickstart/examples/inference-scheduling/README.md

```diff
@@ -13,18 +13,29 @@ This profile defaults to the approximate prefix cache aware scorer, which only o
 1. Install the dependencies; see [install-deps.sh](../../install-deps.sh)
 2. Use the quickstart to deploy Gateway CRDs + Gateway provider + Infra chart. This example uses `kgateway` but should work with `istio` given some modifications as described below step 3.
 
+```bash
+# Set common environment variables
+export NAMESPACE=llm-d-inference-scheduling
+export GATEWAY=kgateway # choose between [gke-l7-regional-external-managed, istio, kgateway]
+# Set the provider.name in the gaie chart.
+export GAIE_GATEWAY_PROVIDER=$([ "$GATEWAY" = "gke-l7-regional-external-managed" ] && echo "gke" || echo "none")
+```
+
 ```bash
 # From the repo root
 cd quickstart
-HF_TOKEN=$(HFTOKEN) ./llmd-infra-installer.sh --namespace llm-d-inference-scheduling -r infra-inference-scheduling --gateway kgateway
+export HF_TOKEN=$(YOUR_TOKEN)
+./llmd-infra-installer.sh --namespace ${NAMESPACE} -r infra-inference-scheduling --gateway ${GATEWAY}
 ```
     - It should be noted release name `infra-inference-scheduling` is important here, because it matches up with pre-built values files used in this example.
 
 3. Use the helmfile to apply the modelservice and GIE charts on top of it.
 
 ```bash
 cd examples/inference-scheduling
-helmfile --selector managedBy=helmfile apply helmfile.yaml --skip-diff-on-install
+helmfile --namespace ${NAMESPACE} --selector managedBy=helmfile \
+--set provider.name=${GAIE_GATEWAY_PROVIDER} \
+apply helmfile.yaml --skip-diff-on-install
```

Comment:
These examples are opinionated - i have no issue with including minimal documentation to explain workarounds for specific providers but I don't want to re-write all the verbiage to be generic.

Reply:
Ok, another acceptable path would be a gke.md file where we could keep all that stuff together - if you did something like that you could add as much or little as you want